src/tiled/layerdock.cpp

```diff
@@ -453,6 +453,7 @@ void LayerView::contextMenuEvent(QContextMenuEvent *event)
         menu.addAction(handler->actionMergeLayersDown());
         menu.addAction(handler->actionRemoveLayers());
         menu.addSeparator();
+        menu.addAction(handler->actionSelectAllLayers());
```

Comment:
It seems to me like it could be useful if this action showed up even if the clicked index is not valid. To keep it in the same position in the menu, I'd suggest to end the `if` block here, and just check `proxyIndex.isValid()` again for the rest of the menu.

Reply:
@bjorn done!  Feel free to remove the comment if you wish - just a heads up in case anyone is wondering why there are two blocks for the same condition